src/hint_processor/builtin_hint_processor/math_utils.rs

```diff
@@ -565,6 +567,66 @@ pub fn assert_250_bit(
     insert_value_from_var_name("low", low, vm, ids_data, ap_tracking)
 }
 
+// Implements hint:
+// %{ ids.is_250 = 1 if ids.addr < 2**250 else 0 %}
+pub fn is_250_bits(
+    vm: &mut VirtualMachine,
+    ids_data: &HashMap<String, HintReference>,
+    ap_tracking: &ApTracking,
+) -> Result<(), HintError> {
+    let addr = get_integer_from_var_name("addr", vm, ids_data, ap_tracking)?;
+
+    // Main logic: ids.is_250 = 1 if ids.addr < 2**250 else 0
+    let is_250 = Felt252::from((addr.as_ref().bits() <= 250) as u8);
+
+    insert_value_from_var_name("is_250", is_250, vm, ids_data, ap_tracking)
+}
+
+/*
+Implements hint:
+%{
+    # Verify the assumptions on the relationship between 2**250, ADDR_BOUND and PRIME.
+    ADDR_BOUND = ids.ADDR_BOUND % PRIME
+    assert (2**250 < ADDR_BOUND <= 2**251) and (2 * 2**250 < PRIME) and (
+            ADDR_BOUND * 2 > PRIME), \
+        'normalize_address() cannot be used with the current constants.'
+    ids.is_small = 1 if ids.addr < ADDR_BOUND else 0
+%}
+*/
+pub fn is_addr_bounded(
+    vm: &mut VirtualMachine,
+    ids_data: &HashMap<String, HintReference>,
+    ap_tracking: &ApTracking,
+    constants: &HashMap<String, Felt252>,
+) -> Result<(), HintError> {
+    let addr = get_integer_from_var_name("addr", vm, ids_data, ap_tracking)?;
+    let prime = Felt252::prime();
+
+    let addr_bound = constants
+        .get(ADDR_BOUND)
+        .ok_or(HintError::MissingConstant(ADDR_BOUND))?
+        .to_biguint()
+        .mod_floor(&prime);
```

Comment:
Constants are already deserialized as `Felt252`, so its not necessary to take modulus prime here

Reply:
Thanks!